### ISSUE TITLE
Run the `MVP` on `localhost`

### ISSUE THREAD

**Issue by nelsonic** [dwyl/mvp#190]
Our objective is to make the **`MVP`** as ultra-beginner-friendly to Devs 
_especially_ those who are **`new`** `Elixir/Phoenix` ...
That means if there are _any_ assumptions about prior knowledge/experience 
we either need to clarify them in _this_ repo or link to where they can learn/understand more
in our _many_ other projects. 🤞 

Following on from #140 @LuchoTurtle please:

+ [x] clone the repo to your `localhost` 💻 
+ [x] #191
  + [x] Get the `AUTH_API_KEY` from https://auth.dwyl.com 🔑 

+ [x] If there are gaps in the `README.md` please:
  + [x] note them in this issue. 💬 
  + [x] create a PR to update them. 🙏 

again, our goal is for _anyone_ to be **up-and-running** with the **`MVP`** in **_less_ than 5 minutes** including `mix setup` and `AUTH_API_KEY` ... so the completeness/quality of the docs is mega important!

# `.then` ...

+ [x] Read through the `BUILDIT.md` file and just sense check to see if we missed anything.
  Again the objective is to give people a _complete_ guide for re-creating the **`MVP`** from scratch.

If you have **_any_ questions** please ask! ❓ 

For clarity, the purpose of this exercise is to familiarise yourself with the **`MVP`** as much as possible.
I will ask you to build a feature for it today. That issue/task will follow shortly. ⏳

**Comment by LuchoTurtle** [comment]
The only "gap" that I discovered was not mentioning the whole `tag` mechanism. I've updated the steps within the README so users go through the tag list and tag their private items. I've also recorded a demo and updated the GIF. 

I had a little detour to the [phoenix-stopwatch](https://github.com/dwyl/phoenix-liveview-stopwatch) repo and noticed it [wasn't building](https://github.com/dwyl/phoenix-liveview-stopwatch/issues/23) while running. So I went ahead and pushed a [PR](https://github.com/dwyl/phoenix-liveview-stopwatch/pull/22) to fix that. 

Onto to the `BUILD.md` file, I'm going to replicate the project as I read 👍
